projects/neoforge/build.gradle

```diff
@@ -222,6 +222,11 @@ generateAccessTransformers {
     )
 }
 
+tasks.withType(JavaCompile.class).configureEach {
+    // Increase memory used during compilation, to avoid OutOfMemoryErrors
+    options.forkOptions.memoryMaximumSize = '2g'
+}
+
```

Comment:
Should we also document the existence of this in `gradle.properties`, in case someone wants to change the Xmx? By the way, the `gradle.properties` can I think be changed back down to `2G`, and the comment can be updated as follows: `decompilation` -> `recompilation`.

Reply:
Done.